covsirphy/ode/sewirf.py

```diff
@@ -41,7 +41,7 @@ class SEWIRF(ModelBase):
             "rho2": 0.167, "rho3": 0.167,
         },
         ModelBase.Y0_DICT: {
-            ModelBase.S: 994_000, ModelBase.E: 3000, ModelBase.W: 0.002,
+            ModelBase.S: 994_000, ModelBase.E: 3000, ModelBase.W: 0,
```

Comment:
Shouldn't W be 2000? In order to N = S+E+W+CI+R+F?

Reply:
Thank you, I will update it to 2000.